index.html

```diff
@@ -161,33 +160,68 @@
             });
             card.appendChild(editButton);
 
-            app.appendChild(card);
-        });
+            const deleteButton = document.createElement("button");
+            deleteButton.textContent = "Delete";
+            deleteButton.classList.add("btn", "waves-effect", "waves-light", "delete-button");
+            deleteButton.innerHTML = '<i class="material-icons">clear</i>';
+            deleteButton.addEventListener("click", () => {
+                app.removeChild(card);
+                timers.splice(timers.indexOf(timer), 1);
+                updateUrlWithTimers(timers);
+            });
+            card.appendChild(deleteButton);
 
-        const scoringSection = document.createElement("div");
-        scoringSection.classList.add("scoring");
+            app.appendChild(card);
+        }
 
-        const scoreLabel = document.createElement("label");
-        scoreLabel.textContent = "Score the list:";
-        scoringSection.appendChild(scoreLabel);
+        function createAddButton() {
+            const addButton = document.createElement("button");
+            addButton.classList.add("btn", "waves-effect", "waves-light", "add-button");
+            addButton.innerHTML = '<i class="material-icons">add</i>';
+            addButton.addEventListener("click", () => {
+                const newTimer = {
+                    description: "New Timer",
```

Comment:
Add the "Delete Timer" button to each card. Place the following code before the editButton.addEventListener("click", () => { line in the "createCard" function: